app/views/burritos/_burrito.html.erb

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<style>
+.checked {
+  color: orange;
+}
+</style>
+</head>
+<body>
+
+<div id="<%= dom_id(burrito) %>" class="card">
+  <h5><%= Restaurant.where({ :id => burrito.restaurant_id }).at(0).name %></h5>
```

Comment:
The `find_by` method is preferred over using `where(...).at(0)`

```rb
Restaurant.find_by({ :id => burrito.restaurant_id }).name
```

If you're looking by `id` column though, you use the `find` method and you don't have to specify the column

```rb
Restaurant.find(burrito.restaurant_id).name
```

But even better, you can use an association again

```suggestion
    <h5><%= burrito.restaurant.name %></h5>
```

what do you think?